docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -425,7 +422,7 @@ For more information, see link:https://quarkus.io/blog/path-resolution-in-quarku
 [[map-security-identity-roles]]
 === Map `SecurityIdentity` roles
 
-Winning role-based policy can map the `SecurityIdentity` roles to the deployment-specific roles.
+When a role-based policy matches and takes precedence ("wins"), it can map the `SecurityIdentity` roles to the deployment-specific roles.
```

Comment:
unlike "winning", here you loose information about "why it takes precedence", can you just say that it happens for the most specific path rule?

Reply:
I like it better. the `rule` is something I used, but it is more like "pattern" and we don't mentioned the "pattern" (we omit it), so let's stick to the `path` please